hathor/simulator/simulator.py

```diff
@@ -151,7 +152,8 @@ def create_peer(
         soft_voided_tx_ids: Optional[Set[bytes]] = None,
         full_verification: bool = True,
         event_ws_factory: Optional[EventWebsocketFactory] = None,
-        tx_storage: Optional[TransactionStorage] = None
+        tx_storage: Optional[TransactionStorage] = None,
+        event_storage: Optional[EventStorage] = None,
```

Comment:
Why `event_storage` instead of `EventManager`?

Reply:
This has been removed after rebasing

Reply:
Ok. Thanks!